src/index.html

```diff
@@ -6,9 +6,46 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img src="images/logo.png" alt="MOYO">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link">
+            <a class="is-active" href="link-1">APPLE</a>
```

Comment:
1. use lowercase at html and you can change this case at css
2. if you have no links you can just use `#` at `href`
3. add a class to a link
```suggestion
            <a class="nav__link is-active" href="#apple">apple</a>
```